docs/quantum_oscilator/architecture.md

```diff
@@ -0,0 +1,81 @@
+# Architecture of Neutal Network
+
+## Parent classes
+
+[`#!python class tf.keras.Model`](https://keras.io/api/models/model/)
+
+## Introduction
+
+QONetwork class is responsible for the learning process from the user input.
+Our neural network architecture that simultaneously learns eigenvalues and the
+associated eigenfunctions using a scanning mechanism. The proposed technique is
+an extension to neural network differential equation solvers and, consequently,
+acquires all the benefits that network solvers have over numerical integrators.
+
+### Network structure
+
+| name                                                                                                | type               | shape |
+| --------------------------------------------------------------------------------------------------- | ------------------ | ----- |
+| [`input_1 ` `(InputLayer) `](https://www.tensorflow.org/api_docs/python/tf/keras/layers/InputLayer) | `#!python float32` | `1`   |
+| `eigenvalue ` `(Eigenvalue)`                                                                        | `#!python float32` | `1`   |
+| [`concatenate ` `(Cocatenate)`](https://keras.io/api/layers/merging_layers/concatenate/)            | `#!python float32` | `2`   |
+| [`dense_1 ` `(Dense)`](https://keras.io/api/layers/core_layers/dense/)                              | `#!python float32` | `2`   |
+| [`dense_2 ` `(Dense)`](https://keras.io/api/layers/core_layers/dense/)                              | `#!python float32` | `50`  |
+| [`predictions ` `(Dense)`](https://keras.io/api/layers/core_layers/dense/)                          | `#!python float32` | `1`   |
+
+### Abstract
+
+The construction of the network involves the definition of layers in the
+assemble_hook method. In neural networks there is one input layer (input), one
+output layer (output) and many hidden layers (d1, d2, ...). Adding hidden
+layers involves adding consecutive lines of code in the form dX =
+keras.layers.Dense(number_neurons, activation function, layer_name, number
+precision)(layer_previous).
+
+When building a neural network architecture, parameters within each layer can
```

Comment:
It's not true in this case. Our network has pre defined activation functions. Only number of neurons in internal layer can be changes with QOConstants